apps/wpcom-block-editor/src/wpcom/features/domain-upsell-callout.js

```diff
@@ -0,0 +1,91 @@
+import { getCalypsoUrl } from '@automattic/calypso-url';
+import { dispatch, select, subscribe } from '@wordpress/data';
+import domReady from '@wordpress/dom-ready';
+import { __ } from '@wordpress/i18n';
+import { close, globe, Icon } from '@wordpress/icons';
+import { createElement } from 'react';
+import { render } from 'react-dom';
+import tracksRecordEvent from './tracking/track-record-event';
+
+import './domain-upsell-callout.scss';
+
+function whenEditorIsReady() {
+	return new Promise( ( resolve ) => {
+		const unsubscribe = subscribe( () => {
+			// This will trigger after the initial render blocking, before the window load event
```

Comment:
It's not obvious from the code why to wait for the editor, instead of [`domReady`](https://github.com/WordPress/gutenberg/tree/trunk/packages/dom-ready#readme). Adding a comment might clarify it for future devs.